common/buildcraft/compat/module/ic2/CompatModuleIndustrialCraft2.java

```diff
@@ -0,0 +1,31 @@
+package buildcraft.compat.module.ic2;
```

Comment:
Sorry to be a pain, but could you use the same license header for both of these files as in the main BC repo? (Essentially so it's all licensed under MPLv2)

Reply:
Done. There is no license on other files, so I thought it is because of some reason...